src/OpenApi/Serializer/DocumentationNormalizer.php

```diff
@@ -0,0 +1,443 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\OpenApi\Serializer;
```

Comment:
I added a namespace but they could all live in `ApiPlatform\Core\Swagger` if you prefer and be named V2 and V3 with an alias to the old DocumentationNormalizer classname to avoid BC or something like that